Editor/SettingsAnalysis/iTextureAnalyzer.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using Unity.ProjectAuditor.Editor.Core;
+using UnityEditor;
+
+namespace Unity.ProjectAuditor.Editor.SettingsAnalysis
+{
+    public interface ITextureAnalyzer
+    {
+        void Initialize(ProjectAuditorModule module);
+
+        IEnumerable<ProjectIssue> Analyze(BuildTarget platform);
```

Comment:
I think this will need to take at least a `TextureImporter` as a parameter

Reply:
Wrote in the parameters.